Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt

```diff
@@ -70,11 +74,22 @@ fun ReplyInboxScreen(
     val emailLazyListState = rememberLazyListState()
 
     if (contentType == ReplyContentType.DUAL_PANE) {
-        ReplyDualPaneContent(
-            replyHomeUIState = replyHomeUIState,
-            emailLazyListState = emailLazyListState,
-            modifier = modifier.fillMaxSize(),
-            navigateToDetail = navigateToDetail
+        TwoPane(
+            first = {
+                ReplyEmailList(
+                    emails = replyHomeUIState.emails,
+                    emailLazyListState = emailLazyListState,
+                    navigateToDetail = navigateToDetail
+                )
+            },
+            second = {
+                ReplyEmailDetail(
+                    email = replyHomeUIState.selectedEmail ?: replyHomeUIState.emails.first(),
+                    isFullScreen = false
+                )
+             },
+            strategy = HorizontalTwoPaneStrategy(splitFraction = 0.5f, gapWidth = 16.dp),
+            displayFeatures = calculateDisplayFeatures(activity = LocalContext.current as Activity)
```

Comment:
@alexvanyo is this a way to access display features inside compose or it has to be pass down from main activity ?

Reply:
I think passing it down would be best, `LocalContext.current as Activity` might work here, but could break if `LocalContext` was a context wrapper, and also makes it harder to test (it'd be easier to explicitly pass in a list)